functions/player/fn_endMission.sqf

```diff
@@ -3,7 +3,7 @@
 *   executed on player via init.sqf
 */
 
-mcd_fnc_endMission = {
+endgame_fnc_endMission = {
```

Comment:
Hier überschreibt die Funktion sich selbst. Im Prinzip ist alles was hier in den geschweiften Klammern steht die eigentliche Funktion und die publicVariableEventHandlers gehören in die initPlayerLocal oder sowas.